ecosystem/utils/utils.py

```diff
@@ -37,4 +38,8 @@ def set_actions_output(outputs: List[Tuple[str, Union[str, bool, float, int]]])
     """
     for name, value in outputs:
         logger.info("Setting output variable %s: %s", name, value)
-        print("::set-output name={name}::{value}".format(name=name, value=value))
+        if "GITHUB_OUTPUT" in os.environ:
+            with open(os.environ["GITHUB_OUTPUT"], "a") as github_env:
+                print("{name}={value}".format(name=name, value=value), file=github_env)
```

Comment:
More idiomatic

```suggestion
                github_env.write(f"{name}={value}\n")
```

(print adds a `\n`, so we need to recreate that)